shared/teams/render.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import * as React from 'react'
+import {ClickableBox, Box, Icon, ScrollView, Text} from '../common-adapters'
+import {globalColors, globalStyles} from '../styles'
+
+import type {Props} from './render'
+
+function TeamsRender(props: Props) {
+  // TODO: Find right icon for "Join a team" button.
```

Comment:
iconfont-team-join

Reply:
small nit we tend to make these arrow functions
```const TeamsRender = (props: Props) => ( <Box```

Reply:
Done.